polyfill/index.d.ts

```diff
@@ -100,6 +100,12 @@ export namespace Temporal {
     largestUnit: T;
   }
 
+  export interface RoundOptions<T extends string> {
+    smallestUnit: T;
```

Comment:
@ptomato - Why isn't `largestUnit` also in these options?

Sorry for being late on the draw with this review. You work fast!

Reply:
Oh, never mind: there's no such thing as `largestUnit` for the `round` methods of non-Duration types, because you can't remove larger units on those types like you can with Duration.

Reply:
I suppose I would tend to leave it up for longer if we weren't in a hurry to get everything specified ... but don't hesitate to leave comments on already-closed PRs.

Reply:
Nah, that's fine. Speed is good. We can always fix later if there's a real problem. Plus good to incentivize quick reviews!